teambot.py

```diff
@@ -7,17 +7,21 @@
 import atexit
```

Comment:
:). I added it along with some other paranoia about flushing the DB after losing the entire database. Lesson learned :).